tox.ini

```diff
@@ -7,16 +7,27 @@ envlist =
 requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
+    tox-pypi-filter >= 0.10
+    # See https://github.com/astropy/astropy/pull/10045 for the following pin
```

Comment:
Would it not be better to reference the tox issue here?

Reply:
Yes it was a typo - thanks!